webpack/assets/javascripts/react_app/redux/actions/common/forms.js

```diff
@@ -100,7 +101,7 @@ export const submitForm = ({
         headers,
         params,
         actionTypes,
-        handleError,
+        handleError: handleError || defaultHandleError,
```

Comment:
Why would it?.. AFAIK Javascript also has `short-circuiting`, so if `handleError` was provided then it will be used even though `defaultHandleError` is defined. Also, if `handleError` is `null` (not provided) then the default one will be used. I don't see how they both are going to be used. Or did you mean something different?

Reply:
I meant to ask if we should instead of picking just one error handling run both (when available) or would the defaultHandleError interfere with the custom one?

Reply:
Well, I'd be surprised if I'd provide my own handler, but would keep getting some side effects from the default one. If I'd want to run both, I could simply "import" the default one and include it in my own handler, which I'd then pass in props.

The only reason I'm adding this is to suppress "digging in `errors` response object" since in my case there isn't any. I could use any workaround as well. Something like https://github.com/theforeman/foreman/pull/9665/files#diff-72197facc6157dcf3cfb43837d5bd5ac52939f1b39f1ceece1b7c1a1c2c5e18fR31 would do for my use case as well.

Reply:
Yeah makes sense, the digging in error response should be probably done in a different pr as its not directly fixing this prs topic